scripts/service_enable.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
+set -euxo pipefail
 
 service_name="gatestext.service"
 
-if systemctl is-active $service_name | grep -q "inactive"; then
```

Comment:
Technically the `grep -q active` will capture both `active` and `inactive` states.

As per systemd docs, both are valid unit states - 

>The ACTIVE columns shows the general unit state, one of active, reloading, inactive, failed, activating, deactivating.

Suggestion: The exit-code of `systemctl is-active` will be non-0 if the unit is not active. Maybe just rely on the exit code instead of grep? 

> is-active PATTERN…
Check whether any of the specified units are active (i.e. running). Returns an exit code 0 if at least one is active, or non-zero otherwise. 
https://www.freedesktop.org/software/systemd/man/systemctl.html#is-active%20PATTERN%E2%80%A6

Reply:
Honestly we had `||` condition, if first part returns non zero code then it runs second part. so based on that it won't run service to start if unit is active since exit code is zero

Even non-0 check is not required if I include with `||`.

So I'll update like 
`systemctl is-active $service_name || systemctl start $service_name
`

Reply:
this looks good, thanks!